centreon/tests/e2e/features/Cloud-notifications/05-notification-get.feature

```diff
@@ -0,0 +1,23 @@
+Feature: List Notification Rules
+  As a Centreon user with access to the Notification Rules page
+  I want to view the list of notification rules
+  So that I can effectively organize my alerts in the monitoring system
+
+  Background:
+    Given a user with access to the Notification Rules page
+    And the user is on the Notification Rules page
+    And there is at least one minimum notification rule
+    And pagination is set to 10 items per page
+
+  Scenario Outline: Listing Notification Rules with Pagination
+    When I am on listing page <page>
+    And the result shows <count> items
+    Then I should see a previous page link <previous_page> and next page link <next_page>
```

Comment:
**should** is more suited to being used in specifications rather than being used in gherkin.

Reply:
I don't agree, the pagination test is important. We need to be able to access all the results, and this type of operation makes the test more dynamic with values in a table.